tests/test_replication_key.py

```diff
@@ -3,76 +3,59 @@
 import json
 
 import pendulum
-import pytest
+import sqlalchemy
 from faker import Faker
-from singer_sdk.testing import get_standard_tap_tests
+from singer_sdk.testing.templates import TapTestTemplate
 from sqlalchemy import Column, DateTime, Integer, MetaData, String, Table
 
 from tap_postgres.tap import TapPostgres
 
+TABLE_NAME = "test_replication_key"
 SAMPLE_CONFIG = {
     "start_date": pendulum.datetime(2022, 11, 1).to_iso8601_string(),
     "sqlalchemy_url": "postgresql://postgres:postgres@localhost:5432/postgres",
 }
 
 
-# Run standard built-in tap tests from the SDK:
-def test_standard_tap_tests():
-    """Run standard tap tests from the SDK."""
-    tests = get_standard_tap_tests(TapPostgres, config=SAMPLE_CONFIG)
-    for test in tests:
-        test()
+def setup_test_table(table_name, sqlalchemy_url):
```

Comment:
This doesn't seem specific to test_replication_key, can we move this to `test_core`?

Reply:
Not super important feel free to just make a seperate issue for this if you don't want to refactor now.

Reply:
Yeah I think that makes sense. My original thinking was: It's only used for replication_key right now and I thought it might need a refactor if it gets used in other tests (depending on what those tests need)... 

But yeah I agree with you. Plus they're getting used as resources on that one main test runner that handles everything so makes sense to put em in the same file.

Reply:
I get it, probably not even worth asking you to do! Thanks for doing it anyway :) merged